dxr/testing.py

```diff
@@ -267,11 +265,12 @@ def config_input(cls, config_dir_path):
 
     @classmethod
     def teardown_class(cls):
-        if cls.should_delete_instance:
+        if cls.stop_for_interaction:
+            print "Pausing for interaction at 0.0.0.0:8000..."
```

Comment:
We should wrap this whole thing in an `except KeyboardInterrupt` that catches ctrl-C and then cleans up the indices before exiting. I just noticed that, if an index is referenced in the catalog index but it isn't in the config file, then we throw 500s when trying to view files from it, since config.trees[the tree] in _browse_file fails. (We should also harden that, but that's another bug.)